node/beacon-chain-handler/src/importer.rs

```diff
@@ -31,25 +35,30 @@ pub fn create_beacon_block_importer_task(
     }).and_then(|_| Ok(()))
 }
 
-pub struct BeaconBlockImporter {
-    beacon_chain: Arc<BlockChain<BeaconBlock>>,
+pub struct BlockImporter {
+    beacon_chain: Arc<BeaconBlockChain>,
```

Comment:
if we want to keep separation between shard and beacon chain, i think that it's reasonable to maintain separate importers.

Reply:
i guess after looking into this more.. we are going to have different mechanisms for shard chain consensus and communication so faking that won't really help us out much.